ca/server.go

```diff
@@ -32,6 +34,45 @@ type APISecurityConfigUpdater interface {
 
 var _ APISecurityConfigUpdater = &Server{}
 
+// ExternalCATLSUpdater is a way to notify the server of TLS updates for the external CA
+type ExternalCATLSUpdater interface {
+	NotifyTLSChange() <-chan events.Event
+	GetCertificates() []tls.Certificate
+}
+
+// SecurityConfigTLSUpdater takes a security config and sets a watch so we can tell when TLS credentials have changed
+type SecurityConfigTLSUpdater struct {
```

Comment:
I was just asking if we need a `SecurityConfigTLSUpdater` rather than having the single place where it's used call the `Watch` method directly.